src/_stylesheets/_section-highlight.scss

```diff
@@ -37,3 +37,16 @@
 .app-section-highlight--light-grey {
   background-color: govuk-colour('light-grey');
 }
+
+.app-section-highlight__wrapper--logo {
+  text-align: center;
+
+  img {
+    max-width: 60%;
```

Comment:
Pondering if this is making the wordmark a bit too small on mobile screens. Very much an opinion though, and a non-blocking one at that.

Reply:
Got feedback from @hazalarpalikli that it's fine so I'm gonna merge away